src/description.jl

```diff
@@ -1,19 +1,19 @@
 description(i::AnchoredInterval{P}) where P = description(i, P > zero(P) ? "B" : "E")
 
-function description(interval::AnchoredInterval{P, T}, s::String) where {P, T}
+function description(interval::AnchoredInterval{P,T,L,R}, s::String) where {P,T,L,R}
     return string(
-        first(inclusivity(interval)) ? '[' : '(',
+        L === Closed ? '[' : '(',
         description(anchor(interval), abs(P), s),
-        last(inclusivity(interval)) ? ']' : ')',
+        R === Closed ? ']' : ')',
     )
 end
 
-function description(interval::AnchoredInterval{P, ZonedDateTime}, s::String) where P
+function description(interval::AnchoredInterval{P, ZonedDateTime, L, R}, s::String) where {P,L,R}
```

Comment:
Do we only remove spaces after commas in type lists when all the types are single chars?

Reply:
Yes, for "simple" parametric types I've removed the spaces as otherwise some lines would need to be broken up more. It's not a standard convention of ours but it works pretty well for this package.